content/learn/tutorial.md

```diff
@@ -10,135 +10,158 @@ As you get started with Quire, it is important to familiarize yourself with the
 
 {{< q-figure id="text-editor" >}}
 
-## 5. Enter Publication Metadata
+### Try it for Yourself
 
-The metadata for your publication (its title, subtitle, contributors, publication date, etc.) is used in various areas of your site. It’s used under the hood for {{< q-def "search engine optimization" >}} (SEO), as well as on the site itself in headings, navigation labels, and on your About or Copyright page.
+1. If you haven't already, download and install the text editor of your choosing.
+2. Open your text editor.
+3. Click "File" at the top of your screen and navigate to your home directory.
+4. Select your `my-project` directory and open it in your text editor.
 
-All publication metadata is in the `publication.yml` file for your project, which you’ll find in the `data` directory.
-
-Open the `publication.yml` file and try changing the title and subtitle, saving the changes, and then looking at the preview running in your browser. You’ll see that the cover is updated with your new title. It’s also updated in the top navigation bar and in the expandable menu on the site. As much as possible, Quire works on the principle of having content exist only in one place in your files and using code to display it in multiple places in the publication as needed. This means when you make a change to something you only do it once and it changes everywhere.
+## 5. Enter Publication Metadata
 
-{{< q-figure id="quire-starter-metadata-change" >}}
+The metadata for your publication (its title, subtitle, contributors, publication date, etc.) is used under the hood for {{< q-def "search engine optimization" >}} (SEO). It is also used in various areas of your site including headings, navigation labels, and on your About or Copyright page.  As much as possible, Quire works on the principle of having information exist in only one place in your files and using code to display it in multiple places in the publication as needed. This means when you make a change to something you only do it once and it changes everywhere.
 
-The format of this metadata is called {{< q-def "YAML" >}} (*yam-ul*). It’s designed to be a plain-text way of capturing data. The general principal is to have the name of a data item, followed by a colon, a space, and then the data item’s value. A key-value pair.
+All publication metadata is in the `publication.yaml` file for your project, which you’ll find in the `_data` folder in your `content` directory. The format of how the metadata is stored is called {{< q-def "YAML" >}} (*yam-ul*). It’s designed to be a plain-text way of capturing the type of important that appears in multiple places of your project and is therefore treated as data. The general principal is to have the name of a data item followed by a colon, a space, and then the data item’s value. For example:
```

Comment:
> All publication metadata is in the `publication.yaml` file for your project, which you’ll find in the `_data` folder in your `content` directory.

Normally I'd advocate that we could simplify this as:

> All publication metadata is in the `content/_data/publication.yaml` file.

But perhaps in this context, it's better to really lay it out explicitly?

Reply:
Yeah I think it's better to be explicit in the tutorial, but elsewhere in the docs I'll be sure to use the / convention.